<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1378</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1256</width>
    <height>870</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/habib/.designer/backup/Deliverables/downloads.png</normaloff>C:/Users/habib/.designer/backup/Deliverables/downloads.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(42, 42, 42);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="fiveg">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(24, 24, 24);
color: rgb(255, 255, 255);
border: 1px;
border-radius: 15px;
font-weight:800;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Deliverables/5g.png</normaloff>Deliverables/5g.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ultrasound">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(24, 24, 24);
color: rgb(255, 255, 255);
border: 1px;
border-radius: 15px;
font-weight:800;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Deliverables/ultrasound.png</normaloff>Deliverables/ultrasound.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tumor">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(24, 24, 24);
color: rgb(255, 255, 255);
border: 1px;
border-radius: 15px;
font-weight:800;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Deliverables/tumor.png</normaloff>Deliverables/tumor.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="transmitter_radioButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton {
    font: bold 14px;           /* Font size and weight */
    color: #cc3c3c;           /* Text color */
    spacing: 8px;            /* Space between the button and text */
	border: transparent;
}

QRadioButton::indicator {
    width: 14px;              /* Size of the radio circle */
    height: 14px;
}

QRadioButton::indicator::unchecked {
    border: 2px solid #cc3c3c; /* Border of the unchecked circle */
    background-color: white;
    border-radius: 7px;      /* Make it a perfect circle */
}

QRadioButton::indicator::checked {
    border: 2px solid  #cc3c3c;
    background-color:  #cc3c3c;
    border-radius: 7px;
}
</string>
            </property>
            <property name="text">
             <string>Transmitter </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="reciever_radioButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton {
    font: bold 14px;           /* Font size and weight */
    color: #cc3c3c;           /* Text color */
    spacing: 8px;            /* Space between the button and text */
	border: transparent;
}

QRadioButton::indicator {
    width: 14px;              /* Size of the radio circle */
    height: 14px;
}

QRadioButton::indicator::unchecked {
    border: 2px solid #cc3c3c; /* Border of the unchecked circle */
    background-color: white;
    border-radius: 7px;      /* Make it a perfect circle */
}

QRadioButton::indicator::checked {
    border: 2px solid  #cc3c3c;
    background-color:  #cc3c3c;
    border-radius: 7px;
}
</string>
            </property>
            <property name="text">
             <string>Reciever</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Number of Antennas:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="Transmitter_slider">
          <property name="styleSheet">
           <string notr="true">QSlider {
    border: none;
}
QSlider::groove:horizontal {
    height: 5px;
    background: white;  
    border-radius: 3px;   
}

QSlider::add-page:horizontal {
    background: #cc3c3c;
    border: 1px solid #cc3c3c;
    height: 3px;
    border-radius: 1.5px;
}

QSlider::handle:horizontal {
    background: #ffffff;  
    border: 2px solid #cc3c3c;
    width: 10px;
    height: 10px;
    border-radius: 6px;    
    margin: -4px 0;       
}

</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="transmitter_label">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#ffffff</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Array Geometry:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="arraygeometry">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
                color: rgb(255, 255, 255);
                font-size: 18px;
                background-color: rgb(24, 24, 24);
                padding-left: 15px;
                border: 1px solid transparent;
                border-radius: 8px; /* Rounded corners */
            }

            QComboBox QAbstractItemView {
                background-color: #444444;    /* Dropdown list background */
                color: #ffffff;               /* Dropdown list text color */
                selection-background-color: #555555;  /* Highlight background */
                selection-color: #FF5757;     /* Highlighted text color */
            }

            /* Remove the default arrow */
            QComboBox::drop-down {
                margin-right: 10px;
                border-top-right-radius: 15px; /* Apply radius to the top-right */
                border-bottom-right-radius: 15px; /* Apply radius to the bottom-right */
            }

            /* Customize the arrow (triangle) */
            QComboBox::down-arrow {
                image: url(Deliverables/down-arrow.png); /* Optional: use a custom image for the arrow */
                width: 10px;
                height: 10px;
                margin-right: 10px; /* Moves the arrow more to the right */
            }&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Radius of Curvature (For Curved Array):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSlider" name="roc_slider">
          <property name="styleSheet">
           <string notr="true">QSlider {
    border: none;
}
QSlider::groove:horizontal {
    height: 5px;
    background: white;  
    border-radius: 3px;   
}

QSlider::add-page:horizontal {
    background: #cc3c3c;
    border: 1px solid #cc3c3c;
    height: 3px;
    border-radius: 1.5px;
}

QSlider::handle:horizontal {
    background: #ffffff;  
    border: 2px solid #cc3c3c;
    width: 10px;
    height: 10px;
    border-radius: 6px;    
    margin: -4px 0;       
}

</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #ffffff</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSlider" name="frequency_slider">
          <property name="styleSheet">
           <string notr="true">QSlider {
    border: none;
}
QSlider::groove:horizontal {
    height: 5px;
    background: white;  
    border-radius: 3px;   
}

QSlider::add-page:horizontal {
    background: #cc3c3c;
    border: 1px solid #cc3c3c;
    height: 3px;
    border-radius: 1.5px;
}

QSlider::handle:horizontal {
    background: #ffffff;  
    border: 2px solid #cc3c3c;
    width: 10px;
    height: 10px;
    border-radius: 6px;    
    margin: -4px 0;       
}

</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>MHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Shift:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSlider" name="shift_slider">
          <property name="styleSheet">
           <string notr="true">QSlider {
    border: none;
}
QSlider::groove:horizontal {
    height: 5px;
    background: white;  
    border-radius: 3px;   
}

QSlider::add-page:horizontal {
    background: #cc3c3c;
    border: 1px solid #cc3c3c;
    height: 3px;
    border-radius: 1.5px;
}

QSlider::handle:horizontal {
    background: #ffffff;  
    border: 2px solid #cc3c3c;
    width: 10px;
    height: 10px;
    border-radius: 6px;    
    margin: -4px 0;       
}

</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="styleSheet">
           <string notr="true">color:white;
font-size: 10px;
font-weight:600;
border-color: transparent;</string>
          </property>
          <property name="text">
           <string>°</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>BeamForming Map:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="beamforming_map" native="true">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(24, 24, 24);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight:600;
border-color: transparent;</string>
        </property>
        <property name="text">
         <string>Beam Profile:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="beam_profile" native="true">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(24, 24, 24);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
